src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,42 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
+
 public class WorkWithFile {
+    private static final String SEPARATOR = "\\s*,\\s*";
+    private static final int TYPE = 0;
+    private static final int SIZE = 1;
+
     public void getStatistic(String fromFileName, String toFileName) {
```

Comment:
To improve code structure and readability, consider breaking down the process into three distinct methods: readFile(), generateReport(), and writeToFile(). Each method will have a clear responsibility, enhancing maintainability. For instance:

1. Use readFile(fromFileName) to fetch and return the file's contents.
2. Pass the content to generateReport(data) to create the desired report.
3. Finally, call writeToFile(toFileName, report) to save the report.

Applying this structure to your current code would not only simplify the getStatistic method but also follow good coding practices. Here's a quick outline:

    String data = readFile(fromFileName);
    String report = generateReport(data);
    writeToFile(toFileName, report);

This approach will keep your code organized and make each part easier to test and debug.